### PR TITLE
Feat: add cycle sort

### PR DESCRIPTION
**Pull request by BSzmolke**
### Description

Add short description of cycle sort algorithm

### Checklist

- [x] A description of the changes in this PR is mentioned above.
- [x] All the new and existing checks pass.
- [x] No plagiarized, duplicated, or repetitive documentation that has been directly copied from another source.
- [x] If it's a new explanation, it contains solid, understandable, and accessible information.
- [x] I have read the whole [contributing guidelines](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/CONTRIBUTING.md) and agree to the [Code of Conduct](https://github.com/TheAlgorithms/.github/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by appgurueu — DISMISSED** [review]
Looks fine, minor formatting issues can be fixed later on. Aren't the implementation links redundant with the The Algorithms page?

**Comment by BSzmolke** [comment]
> Looks fine, minor formatting issues can be fixed later on. Aren't the implementation links redundant with the The Algorithms page?

@appgurueu I thought it's expected as per line:

https://github.com/TheAlgorithms/Algorithms-Explanation/blob/d01394019bacefb5bb6da4e37098813c5fbb597e/CONTRIBUTING.md?plain=1#L37

But I'm happy to remove if you think it would be better without it

**Comment by BSzmolke** [comment]
Redundant links removed

**Review by appgurueu — DISMISSED** [review]

File: en/Sorting Algorithms/Cycle Sort.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+#### Video Explanation
+
+[A video explaining the Cycle Sort Algorithm](https://www.youtube.com/watch?v=gZNOM_yMdSQ)
+
+#### The Algorithms Page
+
+[Cycle Sort](https://the-algorithms.com/algorithm/cycle-sort)
+
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Cycle Sort.md:70]:
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

**Pull request merged**